types.d.ts

```diff
@@ -0,0 +1,15 @@
+import type { DefaultUser } from 'next-auth'
```

Comment:
This is some janky code we have to use for next-auth to allow us to access and manipulate user ids

Reply:
Where are we manipulating the ids?